packages/react-form/src/utilities.ts

```diff
@@ -97,3 +97,14 @@ export function validateAll(fieldBag: {[key: string]: FieldOutput<any>}) {
 }
 
 export function noop() {}
+
+/**
+ * Transforms a boolean Field object to work with checkboxes and radios.
+ * @param field
+ */
+export function asChoiceField({value: checked, ...fieldData}: Field<boolean>) {
```

Comment:
could we also expose this as `useChoiceField` where we create the field inline?

Reply:
oooooooo great idea!